daemon/daemon_main.go

```diff
@@ -1419,5 +1419,8 @@ func (d *Daemon) instantiateAPI() *restapi.CiliumAPI {
 	api.PolicyGetFqdnCacheIDHandler = NewGetFqdnCacheIDHandler(d)
 	api.PolicyGetFqdnNamesHandler = NewGetFqdnNamesHandler(d)
 
+	// /ip/cache/
```

Comment:
The API documents `/ip`, the code says `/ip/cache`. It's fixed up in the next commit though so I think it's just a small bug in how you squashed things together.